RxJavaTemplate/build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     ext {
-        build_gradle_version = '4.1.1'
+        build_gradle_version = '4.2.0-rc01'
```

Comment:
I don't think we should consider using beta version, especially for the Template project. The reason for this it's just because we're using beta AS in some project ❌

Reply:
ah okay